src/hooks/api/organizers.ts

```diff
@@ -263,9 +257,9 @@ const useGetOrganizersByCreatorQuery = (
           ? `${creator?.['https://publiq.be/uitidv1id']} OR`
           : ''
       } ${creator?.email}) OR contributors:${creator?.email}`,
-      limit: paginationOptions.limit,
-      start: paginationOptions.start,
-      embed: true,
+      limit: `${paginationOptions.limit}`,
+      start: `${paginationOptions.start}`,
```

Comment:
This is subjective but I find `String(paginationOptions.limit)` clearer than the template string in terms of signifying those need to be cast maybe

Reply:
I rarely use the String, Number, ... functions myself 😄 
Since it is a matter of preference I'll leave it for now